src/src.sln

```diff
@@ -0,0 +1,6 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 2013
```

Comment:
🤣

Reply:
Removed. That was interestingly created by either Visual Studio Code or Rider.